clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutor.scala

```diff
@@ -23,7 +23,9 @@ class DeleteExecutor[CI <: ClioIndex](deleteCommand: DeleteCommand[CI])(
   implicit ec: ExecutionContext
 ) extends Executor {
 
-  private val prettyKey = ClassUtil.formatFields(deleteCommand.key)
+  import deleteCommand.index.implicits._
+
+  private val prettyKey = deleteCommand.key.show
```

Comment:
Does `show` do the same thing as `formatFields()` did? If we had a type `sealed trait IndexCommand(val index:CI) extends ClioCommand` we could add `key` and `prettyKey` to that to avoid having to import `show` here.

Reply:
It doesn't show exactly the same thing, but it shows the same information. The way I've set it up, it'll show snake-case field names instead of camel-case (which is how the outside world sees the fields, so I think it's an improvement).

Reply:
Tried out pulling the key into the command name, and found a concrete benefit of the `show` approach. For commands like `patch` that don't take a single key as input, we can't encode a pretty key within the command object, but we'd still like to be able to use a consistent key->string definition once we've gotten our hands on keys from the server. Backfill is done with now, but the same situation would have applied there.